cmake/linux/package_linux.sh.in

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env bash
+# Creates Linux ".AppImage" for @PROJECT_NAME_UCASE@
+
+set -e
+
+# Console colors
+RED="\\x1B[1;31m"
+GREEN="\\x1B[1;32m"
+YELLOW="\\x1B[1;33m"
+PLAIN="\\x1B[0m"
+
+USERBIN="$HOME/bin"
+LINUXDEPLOYQT="$USERBIN/linuxdeployqt"
+APPDIR="@CMAKE_BINARY_DIR@/@PROJECT_NAME_UCASE@.AppDir/"
+
+# TODO: Since DESTDIR is set at build time, the true install location is unknown.
+INSTALL="@CMAKE_BINARY_DIR@/../target"
+
+# linuxdeployqt assumes installations contain /usr/ prefixes.
+# 1. Make a directory containing /usr
```

Comment:
First, you make mention of `INSTALL_ROOT` but [your wiki](https://github.com/probonopd/linuxdeployqt#using-linuxdeployqt-with-travis-ci) references `DESTDIR`.   Is there a difference?  

```
# quoted from wiki
CMake wants DESTDIR instead:

  - cmake . -DCMAKE_INSTALL_PREFIX=/usr
  - make -j$(nproc)
  - make DESTDIR=appdir install ; find appdir/
```

I did wonder if the parent directory would be created.  We normally use cmake flags, as then we can use them in our scripts as well.  If we can specify this through cmake instead, then we can configure it in and not have to guess. (Problem 2 above).